posthog/models/prompt.py

```diff
@@ -0,0 +1,520 @@
+from typing import Any, Dict, List
+
+from django.db import models
+from django.utils import timezone
+
+
+class PromptSequenceState(models.Model):
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(fields=["team", "person", "key"], name="unique sequence key for person for team")
+        ]
+
+    team: models.ForeignKey = models.ForeignKey("Team", on_delete=models.CASCADE)
+    person: models.ForeignKey = models.ForeignKey("Person", on_delete=models.CASCADE)
```

Comment:
As a user, I don't think it'd make sense for me to go through a flow multiple times just because I have multiple projects. Why not key this just `["person", "key"]`?